pkg/types/builder.go

```diff
@@ -280,6 +280,9 @@ func (g *Builder) buildSchema(sch *schema.Schema, cfg *config.Resource, tfPath [
 				}
 				elemType = paramType
 			}
+		// if unset
```

Comment:
nitpick: A more explanatory comment or link to the issue would be more helpful for future readers.